packages/sdk-codegen/src/sdkModels.spec.ts

```diff
@@ -132,9 +246,9 @@ describe('sdkModels', () => {
         apiTestModel.types.CreateQueryTask
       )
       expect(type).toBeDefined()
-      expect(type!.properties.query_id.required).toEqual(true)
-      expect(type!.properties.result_format.required).toEqual(true)
-      expect(type!.properties.source.required).toEqual(false)
+      expect(type?.properties.query_id.required).toEqual(true)
+      expect(type?.properties.result_format.required).toEqual(true)
+      expect(type?.properties.source.required).toEqual(false)
```

Comment:
does TS complain about `type` possibly being `undefined` inspite of the `toBeDefined` assertion above?

Reply:
yeah, sadly the jest `toBeDefined()` test is not a signal to the Typescript linter.